shared/metrics/src/main/java/io/pravega/shared/Utils.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.shared;
+
+import com.codahale.metrics.Counter;
+import com.codahale.metrics.Meter;
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.Metric;
+import io.pravega.shared.metrics.MetricsProvider;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class Utils {
```

Comment:
This class seems to only be called from tests. So could you please move it to shared test.

Reply:
done. moved to `shared/src/test`.